internal/kubernetes/operator/deployment/deployment_test.go

```diff
@@ -436,71 +438,71 @@ func TestBuildServerlessDeployments(t *testing.T) {
 			},
 		}
 
-		cluster := &mongodbatlas.Cluster{
-			AutoScaling: &mongodbatlas.AutoScaling{
-				AutoIndexingEnabled: pointer.Get(true),
-				Compute: &mongodbatlas.Compute{
-					Enabled:          pointer.Get(true),
-					ScaleDownEnabled: pointer.Get(true),
-					MinInstanceSize:  "M20",
-					MaxInstanceSize:  "M40",
-				},
-				DiskGBEnabled: pointer.Get(true),
-			},
-			BackupEnabled: nil,
-			BiConnector: &mongodbatlas.BiConnector{
-				Enabled:        pointer.Get(true),
-				ReadPreference: "TestRef",
-			},
-			ClusterType:              "SERVERLESS",
-			DiskSizeGB:               pointer.Get[float64](20),
-			EncryptionAtRestProvider: "TestProvider",
-			Labels:                   nil,
-			ID:                       "TestClusterID",
-			GroupID:                  "TestGroupID",
-			MongoDBVersion:           "5.0",
-			MongoDBMajorVersion:      "5.0",
-			MongoURI:                 "",
-			MongoURIUpdated:          "",
-			MongoURIWithOptions:      "",
-			Name:                     clusterName,
-			CreateDate:               "01-01-2021",
-			NumShards:                nil,
-			Paused:                   nil,
-			PitEnabled:               nil,
-			ProviderBackupEnabled:    nil,
-			ProviderSettings: &mongodbatlas.ProviderSettings{
+		cluster := &atlasv2.ServerlessInstanceDescription{
+			// AutoScaling: &mongodbatlas.AutoScaling{
```

Comment:
Should those comments be cleaned up if the code is not needed?

Reply:
makes sense, will clean those up

Reply:
done